holidayextras/ui-toolkit
# Updating stepper to allow new step value

#### What does this PR do? (please provide any background)
This will allow someone to specify a step value to use when incrementing or decrementing the stepper value.
#### What tests does this PR have?
`npm run test`
#### How can this be tested?
Pull down the stepper-stepvalue branch and you can then run the `npm run test` and `npm run coverage` and `npm run lint` in order to check it passes them all for stepper.

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
